php/blocks/class-loader.php

```diff
@@ -177,19 +160,30 @@ protected function editor_assets() {
 		);
 
 		$author_block_slugs = wp_list_pluck( $author_blocks, 'post_name' );
-
-		// Used to conditionally exclude blocks from certain post types.
-		$post      = get_post();
-		$post_type = $post->post_type;
-
 		wp_localize_script(
 			'block-lab-blocks',
 			'blockLab',
 			[
 				'authorBlocks' => $author_block_slugs,
-				'postType'     => $post_type,
+				'postType'     => get_post_type(), // To conditionally exclude blocks from certain post types.
```

Comment:
This is the main edit in the PR. `get_post_type()` will return `false` if `get_post()` returns something falsey. This should prevent a PHP notice.